integration/util/scenario/starport.js

```diff
@@ -202,21 +201,21 @@ async function buildStarport(starportInfo, validatorsInfoHash, ctx, chain, cashT
   } else {
     // Deploy Proxies and Starport
     let proxyAdmin = cashToken.proxyAdmin;
-    let starportImpl = await chain.__deploy('Starport', [cashToken.ethAddress(), chain.root()]);
+    let starportImpl = await chain.__deploy('Starport', [cashToken.ethAddress(), chain.root(), web3.utils.asciiToHex(chain.nameAsUpperCase()), web3.utils.asciiToHex(chain.nameAsStarportHeader())]);
     let proxy = await chain.__deploy('TransparentUpgradeableProxy', [
       starportImpl._address,
       proxyAdmin._address,
       "0x"
-    ], { from: chain.root() });
+    ], {from: chain.root()});
     let starport = await chain.__getContractAt('Starport', proxy._address);
     if (validators.length > 0) {
-      await starport.methods.changeAuthorities(validators).send({ from: chain.root(), gas: 4_000_00 });
+      await starport.methods.changeAuthorities(validators).send({from: chain.root(), gas: 4_000_00});
```

Comment:
Is this gas amount missing a `0`?

Reply:
hmm it does look that way but i actually only accidentally changed whitespace in here, wondering if I should change the underscores or add the zero, for now i'll change the underscore so it doesn't impact functionality (which is working)